### ISSUE TITLE
[bug]: puppeteer Deprecation warning logs

### ISSUE THREAD

**Issue by pankajgupta-web** [jgazeau/website2pdf#101]
### Steps that lead to the issue

old puppeteer version

### Expected behavior

warning logs should not be there 

### Reproducer

When we run PDF application then it starts with below logs : - 

Puppeteer old Headless deprecation warning:

51 In the near feature `headless: true` will default to the new Headless mode

52 for Chrome instead of the old Headless implementation. For more

53 information, please see https://developer.chrome.com/articles/new-headless/.

54 Consider opting in early by passing `headless: "new"` to `puppeteer.launch()`

55 If you encounter any bugs, please report them to https://github.com/puppeteer/puppeteer/issues/new/choose.

Is it possible to suppress these logs ??

**Comment by jgazeau** [comment]
Hi @pankajgupta-web ,

Can you provide more context regarding your issue ?
What is the exact log ? What command are you using ? What versions of the lib are you using ?
I can see these kind of logs on wether the current main branch or the latest 0.0.12 release.

**Comment by pankajgupta-web** [comment]
This is not an issue. These are just warning logs that additionally are coming. Our requirement is if we can suppress these logs. 

These are the starting logs that comes after running below command : - 

npx website2pdf --chromiumFlags="--no-sandbox" --templateDir "./pdf_templates/<templateName>" --displayHeaderFooter --outputDir "./<output_folder_name>"  --exclude-urls="\/categories\/"  --margin-left "27.5px"  --margin-right "27.5px"

We are using latest version (0.0.12) of the website2pdf lib.

**Comment by jgazeau** [comment]
I know it is just warning but as I said I can't reproduce it on the latest 0.0.12 release, so I can't really help you.
Can you provide at least the beggining log of the command you are running within your terminal or pipeline ? Are you using the proper version of puppeteer ?

**Comment by pankajgupta-web** [comment]
We have used below command in Docker file for puppeteer

```dockerfile
# Skip Chromium installation, as it has been done in previous command
ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true \
    PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium-browser
```

Please find below image for complete run command with output of warning. 

![image](https://github.com/jgazeau/website2pdf/assets/96823809/2e3699f3-12c5-42ec-b706-f46d69c58db7)

**Comment by pankajgupta-web** [comment]
Hi Jordan. Is there any update on this ?

**Comment by jgazeau** [comment]
Hi,
Thew things to consider
1. I've added a chromiumHeadless option so mode can be decided on command line
2. By default the mode will remains true. I've noticed some performances impacts by activating the `new` headless mode and there is an issue on puppeteer side (https://github.com/puppeteer/puppeteer/issues/10071)
3. Considering this [issue](https://github.com/argos-ci/jest-puppeteer/issues/553), using the `1` option should prevent your warnings.
4. Using the `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` variable will prevent using the recommended chromium puppeteer version embedded in the website2pdf dependencies. This might cause your issue as it cannot be reproduce when using the version embedded in the website2pdf dependencies. Don't forget to provide these information when creating an issue.

I'll create a new version soon that will integrate the new option
